modules/imgproc/src/smooth.simd.hpp

```diff
@@ -2048,6 +2067,25 @@ void GaussianBlurFixedPoint(const Mat& src, /*const*/ Mat& dst,
     }
 }
 
+void GaussianBlurFixedPoint(const Mat& src, /*const*/ Mat& dst,
```

Comment:
I think it would be better to make a template. You could use `DataType<uint16_t>::depth` to map uint16_t to CV_16U value